javascripts/content_scripts/posting.controller.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * @fileOverview Control the seamless-posting form:
+ * respond to the click of the Privly button.
+ *
```

Comment:
It is getting confusing that there are now different controllers in three different places of the architecture. Privly-chrome, privly-applications, and privly-web all have controllers.